src/main/java/com/coreos/jetcd/data/Header.java

```diff
@@ -9,15 +9,12 @@
   private final long memberId;
   private final long revision;
   private final long raftTerm;
-  private final long compactRevision;
```

Comment:
Maybe I don't change Header to include in this pr since compactRevision doesn't break anything. Once the watch pr is merged then I will have another pr to remove compactRevision.